docs/setup.md

```diff
@@ -0,0 +1,14 @@
+# ORCID auth instructions
+
+#### Steps to obtain ORCID
```

Comment:
Those are already there in default.env, those work fine.
ᐧ

On Wed, Mar 16, 2016 at 1:37 AM, Abigail Cabunoc Mayes <
notifications@github.com> wrote:

> In docs/setup.md
> <https://github.com/mozillascience/PaperBadger/pull/172#discussion_r56231284>
> :
>
> > @@ -0,0 +1,14 @@
> > +# ORCID auth instructions
> > +
> > +#### Steps to obtain ORCID
>
> I meant steps to create an app with ORCID to get the env keys like
> ORCID_AUTH_CLIENT_ID & ORCID_AUTH_CLIENT_SECRET. Did you record that
> anywhere?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mozillascience/PaperBadger/pull/172/files/248d4e4ab866665976f611119f9a395302594fa8#r56231284>
>